### PR TITLE
Introduce Versions.props file to control version of CPS packages

### PR DESCRIPTION
**Pull request by smitpatel**
This file will be used alongwith built version of project-system in C# dev kit for version upgrades

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/9125)

### CODE REVIEW

**Comment by smitpatel** [comment]
Depends on https://devdiv.visualstudio.com/DevDiv/_git/dotnet-project-system-vscode/pullrequest/481012

File: eng/imports/Versions.props [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!-- Any changes to this file or format requires updates in project-system-vscode -->
+<Project>
+  <PropertyGroup>
+    <CPSPackageVersion>17.7.294-pre</CPSPackageVersion>
```
Comment (adamint) [inline eng/imports/Versions.props:4]: Why is this here and not in any of the other props files?

**Review by adamint — APPROVED** [review]

File: eng/imports/HostAgnostic.props [hunk 2]
```diff
@@ -9,7 +9,6 @@
     <!-- Infrastructure -->
     <!-- Path Property: PkgCodecov -->
     <PackageReference Include="Codecov"                             ExcludeAssets="all" GeneratePathProperty="true" />
-    <PackageReference Include="MicroBuild.Core.Sentinel"                  PrivateAssets="all" />
```
Comment (kvenkatrajan) [inline eng/imports/HostAgnostic.props:12]: Why are we removing this reference? What does it do currently?

File: eng/imports/Versions.props (hunk 1 above)
Comment (smitpatel) [reply eng/imports/Versions.props:4]: This file is used in project-system-vscode which adds element for PS.Managed version into it and uses it in other repo. It is separate since it is being used by automation. We can add more items to this file but to keep automation simple, a simple separate file like this is needed.

File: eng/imports/HostAgnostic.props (hunk 2 above)
Comment (smitpatel) [reply eng/imports/HostAgnostic.props:12]: I am not sure what it did but it is not required anymore. We have other MicroBuild packages which brings in whatever we need to build. My assumption at this point is this was needed for microbuild to function few years ago when our build infra was different. (That package was last modified in 2019)
